src/main/java/com/example/gruppajava/entity/Car.java

```diff
@@ -0,0 +1,70 @@
+package com.example.gruppajava.entity;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+
+@JsonIdentityInfo(
+  generator = ObjectIdGenerators.PropertyGenerator.class, 
+  property = "id"
+)
+@Entity(name = "Car")
+@Table(name = "car")
+public class Car {
+  @Id
+  @GeneratedValue(
+    strategy = GenerationType.IDENTITY
+  )
+  private Long id;
```

Comment:
In the rational entity design on Figma, we chose to have a property "model" for the entity Car that I missed in the current code. Shall we ignore it now?